data/json/items/comestibles/egg.json

```
@@ -1169,5 +1151,105 @@
     "material": [ "egg", "veggy" ],
     "fun": 5,
     "flags": [ "EATEN_HOT", "FREEZERBURN" ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_proxy_frog",
```

```suggestion
    "abstract": "egg_proxy_frog",
```

I'm assuming this is supposed to be an abstract?